bitbots_ceiling_cam/config/camera_settings_ceiling_cam.yaml

```diff
@@ -0,0 +1,119 @@
+exposure: 8000.0
+
+#  The target gain in percent of the maximal value the camera supports
+#  For USB-Cameras, the gain is in dB, for GigE-Cameras it is given in so
+#  called 'device specific units'.
+gain: 0.2
+
+#  Gamma correction of pixel intensity.
+#  Adjusts the brightness of the pixel values output by the camera's sensor
+#  to account for a non-linearity in the human perception of brightness or
+#  of the display system (such as CRT).
+gamma: 1.0
+
+#  The average intensity value of the images. It depends the exposure time
+#  as well as the gain setting. If 'exposure' is provided, the interface will
+#  try to reach the desired brightness by only varying the gain. (What may
+#  often fail, because the range of possible exposure vaules is many
+#  times higher than the gain range). If 'gain' is provided, the interface will
+#  try to reach the desired brightness by only varying the exposure time. If
+#  gain AND exposure are given, it is not possible to reach the brightness,
+#  because both are assumed to be fix.
+# brightness: 100
+
+#  Only relevant, if 'brightness' is set:
+#  The brightness_continuous flag controls the auto brightness function.
+#  If it is set to false, the brightness will only be reached once.
+#  Hence changing light conditions lead to changing brightness values.
+#  If it is set to true, the given brightness will be reached continuously,
+#  trying to adapt to changing light conditions. This is only possible for
+#  values in the possible auto range of the pylon API which is e.g. [50 - 205]
+#  for acA2500-14um and acA1920-40gm
+# brightness_continuous: true
+
+#  Only relevant, if 'brightness' is set:
+#  If the camera should try to reach and / or keep the brightness, hence
+#  adapting to changing light conditions, at least one of the following flags
+#  must be set.
+#  If both are set, the interface will use the profile that tries to keep the
+#  gain at minimum to reduce white noise.
+#  The exposure_auto flag indicates, that the desired brightness will be
+#  reached by adapting the exposure time.
+#  The gain_auto flag indicates, that the desired brightness will be
+#  reached by adapting the gain.
+# exposure_auto: true
+# gain_auto: true
+
+##########################################################################
+
+#  The timeout while searching the exposure which is connected to the
+#  desired brightness. For slow system this has to be increased.
+# exposure_search_timeout: 5.0
+
+#  The exposure search can be limited with an upper bound. This is to prevent
+#  very high exposure times and resulting timeouts.
+#  A typical value for this upper bound is ~2000000us.
+# auto_exposure_upper_limit: 2000000.0
+
+#  The MTU size. Only used for GigE cameras.
+#  To prevent lost frames configure the camera has to be configured
+#  with the MTU size the network card supports. A value greater 3000
+#  should be good (1500 for RaspberryPI)
+# gige:
+gige/mtu_size: 1500
```

Comment:
these low settings were necessary for my laptop. it should be documented, that these parameters might be changed

Reply:
having it as default on a low value to make it work on laptops is fine i guess. In my opinion the comment above is enough to explain the parameter. I will add a note in the main readme